lib/account/account-service.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+/**
+ * What we initially send back to the client so they can start to initiate
+ * the complete procedure and save information.
+ *
+ * @typedef {accountService~initiateResult}
+ * @param {string} accountId
+ * @param {string} email
+ * @param {string} mfa
+ * @param {string} salt
+ */
+
+/**
+ * Handles account creation from initiating to confirmation.
+ */
+class AccountService {
+    /**
+     * Used for Dependency Injection and initial configuration
+     * of the storage plugin.
+     *
+     * @param {Object} config
+     * @param {Object} password
+     * @param {Object} storage
+     */
+    constructor(config, password, storage) {
```

Comment:
Config is used on the line 29 to configure the storage plugin. Because configure returns the object, I thought it would be ok to just pass config into the configure method and have it chained on where I do this.storage = storage.